SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -140,6 +140,11 @@
   <Target Name="SonarQubeCategoriseProject"
           Condition=" $(SonarQubeTempPath) != '' ">
 
+    <!-- SONARMSBRU-235: Exclude SQL Server projects which imported Microsoft.Data.Tools.Schema.SqlTasks.targets -->
+    <PropertyGroup>
+      <SonarQubeExclude Condition=" $(SqlTargetName) != '' ">true</SonarQubeExclude>
```

Comment:
I initially wanted to base this on `$(OutputType) == 'Database'`, but the guys who wrote `Microsoft.Data.Tools.Schema.SqlTasks.targets` really hacked to get their stuff to work:

```
<!--
 Define properties that must be set before importing .Net .targets.
 -->
<PropertyGroup>
  <AddAdditionalExplicitAssemblyReferences>false</AddAdditionalExplicitAssemblyReferences>
  <OutputType>Library</OutputType>
  <TargetExt>.dll</TargetExt>
  <!-- ... -->
</PropertyGroup>

...

<!--
 Import the appropriate targets file based on language.
 -->
<Import Condition="'$(TargetLanguage)' == 'CS'" Project="$(MSBuildBinPath)\Microsoft.CSharp.targets" />
<Import Condition="'$(TargetLanguage)' == 'VB'" Project="$(MSBuildBinPath)\Microsoft.VisualBasic.targets" />
```

and then of course the original value for `$OutputType` is lost - it remains set to `Library`